Packs/rasterize/Integrations/rasterize/README.md

```diff
@@ -0,0 +1,201 @@
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Rasterize.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| with_error | Return Errors  | False |
+| wait_time | Time to wait before taking a screen shot \(in seconds\) | False |
+| max_page_load_time | Maximum time to wait for a page to load \(in seconds\) | False |
+| chrome_options | Chrome options \(Advanced. See \[?\]\) | False |
+| proxy | Use system proxy settings | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+**Configuration Notes:**
+* Return Errors: If this checkbox is not selected, a warning will be returned instead of an error entry.
+* Use system proxy settings: Select this checkbox to use the system's proxy settings. **Important**: this integration does not support proxies which require authentication.
+* Chrome options: Add or remove Chrome options used to rasterize. Supports a comma-separated list. If a value contains a comma (for example, when setting the user agent value), escape it with the backslash (**\\**) character. To remove a default option being used, put the option in square brackets. For example, to add the option *--disable-auto-reload* and remove the option *--disable-dev-shm-usage*, set the following value:
+```
+--disable-auto-reload,[--disable-dev-shm-usage]
+```
+
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### rasterize
+***
+Converts the contents of a URL to an image file or a PDF file.
+
+
+#### Base Command
+
+`rasterize`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| wait_time | Time in seconds to wait before taking a screenshot | Optional | 
+| max_page_load_time | Maximum time to wait for a page to load (in seconds) | Optional | 
+| url | The URL to rasterize. Must be the full URL, including the http prefix. | Required | 
+| width | The page width, for example, 1024px. Specify with or without the px suffix. | Optional | 
+| height | The page height, for example, 800px. Specify with or without the px suffix. | Optional | 
+| type | The file type to which to convert the contents of the URL. Can be "pdf" or "png". Default is "png". | Optional | 
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+```!rasterize url=http://google.com```
+
+#### Context Example
+```json
+{
+    "InfoFile": {
+        "EntryID": "922@6e069bc4-2a1e-43ea-8ed3-ea558e377751",
+        "Extension": "png",
+        "Info": "image/png",
+        "Name": "url.png",
+        "Size": 29909,
+        "Type": "PNG image data, 1024 x 800, 8-bit/color RGBA, non-interlaced"
+    }
+}
+```
+
+#### Human Readable Output
```

Comment:
Since this integration generates images let's add here a screen shot from the war room with the site preview.